coordinator/internal/orm/batch.go

```diff
@@ -76,7 +76,7 @@ func (*Batch) TableName() string {
 }
 
 // GetUnassignedBatch retrieves unassigned batch based on the specified limit.
-// The returned batch are sorted in ascending order by their index.
+// The returned batches are sorted in ascending order by their indices.
```

Comment:
this `index` is filed, not indices.

Reply:
got it. all changed to `index fields` in [c96c4dd](https://github.com/scroll-tech/scroll/pull/1411/commits/c96c4dde5822ecb84099d487edbe52cd89299b03).